pandas/core/generic.py

```diff
@@ -2978,15 +2978,54 @@ def add_prefix(self, prefix):
 
     def add_suffix(self, suffix):
         """
-        Concatenate suffix string with panel items names.
+        Suffix labels with string `suffix`.
+
+        For Series, the row labels are suffixed.
+        For DataFrame, the column labels are suffixed.
 
         Parameters
         ----------
-        suffix : string
+        suffix : str
+            The string to add after each label.
 
         Returns
         -------
-        with_suffix : type of caller
+        Series or DataFrame
+            New Series or DataFrame with updated labels.
+
+        See Also
+        --------
+        Series.add_prefix: Prefix labels with string `prefix`.
```

Comment:
As it's for both `Series` and `DataFrame` would make sense to have the `.add_prefix` method of both classes. Or none of them, there is some discussion about whether to have them or not here: https://github.com/pandas-dev/pandas/pull/20289#discussion_r174212593

Reply:
Good call. Added in the latest commit to both.